pkg/routeagent_driver/main.go

```diff
@@ -249,3 +253,14 @@ func waitForNodeReady(k8sClientSet *kubernetes.Clientset) {
 		time.Sleep(1 * time.Second)
 	}
 }
+
+func removeInvalidSockets() {
+	// This can be removed once we stop supporting upgrades from 0.16.0 or older
+	for _, dir := range []string{"/run/openvswitch/db.sock", "/var/run/openvswitch/ovnnb_db.sock", "/var/run/ovn-ic/ovnnb_db.sock"} {
+		info, err := os.Stat(dir)
+		if (err == nil || errors.Is(err, fs.ErrExist)) && info.IsDir() {
+			logger.Infof("Deleting invalid socket %s", dir)
+			os.Remove(dir)
```

Comment:
Do we want to check os.Remove err value?

Reply:
Good point, I’ll rework this to log either success or failure.